packages/smart-wallet/test/test-psm-integration.js

```diff
@@ -155,108 +156,148 @@ test('want stable', async t => {
-  {
-    /** @type {import('../src/invitations.js').ContinuingInvitationSpec} */
-    const invitationSpec = {
-      source: 'continuing',
-      previousOffer: 33,
-      invitationMakerName: 'makeVoteInvitation',
-      invitationArgs: [questionHandle],
-    };
-    /** @type {import('../src/offers').OfferSpec} */
-    const offerSpec = {
-      id: 44,
-      invitationSpec,
-      offerArgs: { positions: [yesFilterOffers] },
-      proposal: {},
-    };
-
-    // wait for the previousOffer result to get into the purse
-    await eventLoopIteration();
-    await offersFacet.executeOffer(offerSpec);
-  }
 
-  t.log('Make sure vote happened');
+  // Call for a vote ////////////////////////////////
+
+  /** @type {import('../src/invitations.js').ContinuingInvitationSpec} */
+  const proposeInvitationSpec = {
+    source: 'continuing',
+    previousOffer: 44,
+    invitationMakerName: 'VoteOnPauseOffers',
+    invitationArgs: harden([psmInstance, ['wantStable'], 2n]),
+  };
+
+  /** @type {import('../src/offers').OfferSpec} */
+  const proposalOfferSpec = {
+    id: 45,
+    invitationSpec: proposeInvitationSpec,
+    proposal: {},
+  };
+
+  await offersFacet.executeOffer(proposalOfferSpec);
   await eventLoopIteration();
-  t.like(computedState.offerStatuses[44], {
-    id: 44,
-    result: { chosen: { strings: ['wantStable'] }, shares: 1n },
-  });
+
+  // vote /////////////////////////
+
+  const committeePublic = E(zoe).getPublicFacet(economicCommittee);
+  const questions = await E(committeePublic).getOpenQuestions();
+  const question = E(committeePublic).getQuestion(questions[0]);
+  const { positions, issue, electionType, questionHandle } = await E(
+    question,
+  ).getDetails();
+  t.is(electionType, 'offer_filter');
+  const yesPosition = harden([positions[0]]);
+  t.deepEqual(issue.strings, ['wantStable']);
+  t.deepEqual(yesPosition, [{ strings: ['wantStable'] }]);
+
+  const voteInvitationDetails = await getInvitationFor(
+    'Voter0',
+    1,
+    computedState.balances,
+  );
+  t.is(voteInvitationDetails.length, 1);
+  const voteInvitationDetail = voteInvitationDetails[0];
+  t.is(voteInvitationDetail.description, 'Voter0');
+  t.is(voteInvitationDetail.instance, economicCommittee);
+
+  /** @type {import('../src/invitations.js').PurseInvitationSpec} */
+  const getCommitteeInvMakersSpec = {
+    source: 'purse',
+    instance: economicCommittee,
+    description: 'Voter0',
+  };
+
+  /** @type {import('../src/offers').OfferSpec} */
+  const committeeInvMakersOffer = {
+    id: 46,
+    invitationSpec: getCommitteeInvMakersSpec,
+    proposal: {},
+  };
+
+  await offersFacet.executeOffer(committeeInvMakersOffer);
+  await eventLoopIteration();
+
+  /** @type {import('../src/invitations.js').ContinuingInvitationSpec} */
+  const getVoteSpec = {
+    source: 'continuing',
+    previousOffer: 46,
+    invitationMakerName: 'makeVoteInvitation',
+    invitationArgs: harden([yesPosition, questionHandle]),
```

Comment:
I was thinking that the `questionHandle` would go here but the `yesPosition` would go in the `offerArgs`; that is: the invitation would be an invitation to vote on this question any way you choose.

But I suppose this makes sense too: the invitation is a filled out ballot, and presenting it an offer is casting it.